.travis.yml

```diff
@@ -2,6 +2,9 @@ language: ruby
 rvm:
   - 2.3.0
 script:
+  - bin/rake db:create RAILS_ENV=test
   - bin/rake db:migrate RAILS_ENV=test
   - bundle exec rspec --color --format documentation
   - bundle exec rubocop
+before_script:
+  - cp config/database.yml.travis config/database.yml
```

Comment:
Since I changed config/database.yml to match the Docker setup (eg host = "db"), this meant that the Travis tests were failing. To fix that I could:
- Use environment variables in database.yml, have those setup differently in Travis and Docker world
- overwrite database.yml for travis
- overwrite database.yml for Docker
- maybe others that I haven't thought of?